core/src/main/python/wlsdeploy/tool/util/mbean_utils.py

```diff
@@ -0,0 +1,767 @@
+"""
+Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+The Universal Permissive License (UPL), Version 1.0
+"""
+import re
+
+import java.lang.Boolean as Boolean
+import java.lang.Exception as JException
+
+from oracle.weblogic.deploy.exception import BundleAwareException
+
+from wlsdeploy.aliases.aliases import Aliases
+from wlsdeploy.aliases.wlst_modes import WlstModes
+from wlsdeploy.exception import exception_helper
+from wlsdeploy.logging.platform_logger import PlatformLogger
+from wlsdeploy.tool.util.alias_helper import AliasHelper
+from wlsdeploy.tool.util.wlst_helper import WlstHelper
+from wlsdeploy.util.weblogic_helper import WebLogicHelper
+
+
+_logger = PlatformLogger('wlsdeploy.mbean.utils')
+
+
+class MBeanUtils(object):
+    """
+    Utility class used to provide information about WLST attributes as retrieved from the MBean MBeanInfo or Interface
+    methods. This class has methods to provide the information stored in different combinations. All methods that want
+    to combine the information from the MBean helpers into different combinations are located in this class.
+    """
+
+    def __init__(self, model_context, exception_type):
+        self.__model_context = model_context
+        self.__exception_type = exception_type
+        self.__aliases = Aliases(self.__model_context, wlst_mode=self.__model_context.get_target_wlst_mode())
+        self.__alias_helper = AliasHelper(self.__aliases, _logger, exception_type)
```

Comment:
I think we should not construct the Aliases object, but pass it in.  It takes some effort to re-build it per WLS version.  Maybe just pass AliasHelper around, it should have all the methods, and the exception type is baked in.